app/views/ngos/registrations/edit.html.erb

```diff
@@ -1,34 +1,29 @@
 <h2><%=  t("devise.registrations.edit.title", resource: t("activerecord.models.ngo.one")) %></h2>
 
-<%=  semantic_form_for(resource, as: resource_name,
-                                 url: registration_path(resource_name),
-                                 html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <%= f.input :email, autofocus: true %>
-  <%= f.input :locale, as: :select, collection: Ngo.locales.keys.to_a %>
-
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Erwarten Bestätigung für: <%= resource.unconfirmed_email %></div>
-  <% end %>
-
-  <%= f.input :password, autocomplete: "off",
-                         hint: "(" + t("devise.registrations.edit.leave_blank_if_you_don_t_want_to_change_it") + ")" %>
-
-  <%= f.input :password_confirmation, autocomplete: "off" %>
-
-  <%= f.input :current_password, autocomplete: "off",
-                                  hint: "(" + t("devise.registrations.edit.we_need_your_current_password_to_confirm_your_changes") + ")" %>
-
-  <%= f.submit t("devise.registrations.edit.update"), class: "btn btn-primary" %>
```

Comment:
Hey @Thrillberg I just checked out your changes - nicely done, thanx a lot 👍 
However, I think you accidentally removed this line. So currently the edit form does not have a submit button 🙈  - could you please add it back in and check that it's working? Thanx a lot!